### PR TITLE
PYT-805 Document Web Servers

### PR DESCRIPTION
**Pull request by hmcginnis**
:pencil: update docs to match each other & include web servers in Python

### CODE REVIEW

File: content/installation/python/PythonSupportedTechnologies.md [hunk 1]
```diff
@@ -1,46 +1,70 @@
-<!-- 
+<!--
 title: "Supported Technologies"
 description: "List of supported technologies"
 tags: "installation Python agent frameworks support package"
 -->
 
-The Python agent supports Python versions 2.7 and 3.5 to 3.8. Framework support is currently for:
+## Contrast agents
```
Comment (danilito19) [inline content/installation/python/PythonSupportedTechnologies.md:7]: I find the generality / generalization? here confusing since we are within docs of Contrast Agents. What about "agent version support"?

File: content/installation/python/PythonSupportedTechnologies.md [hunk 2]
```diff
@@ -1,46 +1,70 @@
...
+
+* [gunicorn](https://gunicorn.org/) 20.0.X
+* [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/#) 2.0.X
+
+## Web frameworks
+
+* [Django](https://www.djangoproject.com/): 1.X & 2.X
+* [Flask](https://flask.palletsprojects.com/en/1.1.x/): 0.10 - 0.12 and 1.X
+* [Pylons](https://pylonsproject.org/): 1.X
+* [Pyramid](https://trypyramid.com/): 1.9
```
Comment (danilito19) [inline content/installation/python/PythonSupportedTechnologies.md:49]: I know this isn't a change for your PR but we could update it now. We also support pyramid 1.4.5 (reddit's version), 1.9, and 1.10.X. Feel free not to add them given that we don't have testing for 1.4.6-1.8.X so we should probably not claim support for those.

File: content/installation/python/PythonSupportedTechnologies.md [hunk 3]
```diff
@@ -1,46 +1,70 @@
...
+* 3.5.X: First supported agent was 1.0.0
+* 2.7.X: First supported agent was 1.0.0
+
+### Deprecated
+
+* 3.4.X: Last supported agent was 1.9.0
+
+## Web servers
+
+* [gunicorn](https://gunicorn.org/) 20.0.X
```
Comment (danilito19) [inline content/installation/python/PythonSupportedTechnologies.md:41]: Since other sections add a preceding paragraph, maybe adding "Many Python frameworks can be run with various web servers. Currently the python agent can be run with ..."

File: content/installation/python/PythonSupportedTechnologies.md [hunk 4]
```diff
@@ -1,46 +1,70 @@
...
-* SQLAlchemy (`SQLAlchemy`)
 * Flask-SQLAlchemy (`Flask-SQLAlchemy`)
+* SQLAlchemy (`SQLAlchemy`)
 
-## OS Support
 
-Agent testing is done on **64-bit OSX** and **64-bit Linux**.
+## Testing environments
+
+When changes are made, Contrast runs a battery of automated tests to ensure that it detects findings in supported technologies across all supported versions of Python.
```
Comment (danilito19) [inline content/installation/python/PythonSupportedTechnologies.md:68]: This doesn't seem related to support, just about how we maintain our agent. I wouldn't personally add this section within the SupportedTechnologies.md

File: content/installation/python/PythonSupportedTechnologies.md (hunk 1 above)
Comment (kmckeedocs) [reply content/installation/python/PythonSupportedTechnologies.md:7]: FYI, this section header will not appear in the new documentation. Headers will read: "Language version" and "Application frameworks"

File: content/installation/python/PythonSupportedTechnologies.md (hunk 2 above)
Comment (kmckeedocs) [reply content/installation/python/PythonSupportedTechnologies.md:49]: Adding these changes to the new documentation. Adding Pyramid 1.4.5 (reddit's version),  and 1.10.X.

File: content/installation/python/PythonSupportedTechnologies.md (hunk 3 above)
Comment (kmckeedocs) [reply content/installation/python/PythonSupportedTechnologies.md:41]: Adding a new row for "Web servers" in the new documentation with this content.

File: content/installation/python/PythonSupportedTechnologies.md (hunk 4 above)
Comment (kmckeedocs) [reply content/installation/python/PythonSupportedTechnologies.md:68]: Agreed. I'd rather gather consistent information about how all of our agents are tested and include that information at a higher level.

File: content/installation/python/PythonSupportedTechnologies.md (hunk 1 above)
Comment (hmcginnis) [reply content/installation/python/PythonSupportedTechnologies.md:7]: Given that it's going away, I'm going to leave as is for now so that it matches other pages.

File: content/installation/python/PythonSupportedTechnologies.md (hunk 2 above)
Comment (hmcginnis) [reply content/installation/python/PythonSupportedTechnologies.md:49]: @danilito19 is there a ticket to add testing for those versions? I think we should add them to screener so that we can claim support.

File: content/installation/python/PythonSupportedTechnologies.md (hunk 4 above)
Comment (hmcginnis) [reply content/installation/python/PythonSupportedTechnologies.md:68]: Both Node and Ruby have it, so I added it here for consistency. I'm happy for it not to make the transition to the new pages, but would like to add it here for consistency in the meantime

File: content/installation/python/PythonSupportedTechnologies.md (hunk 2 above)
Comment (danilito19) [reply content/installation/python/PythonSupportedTechnologies.md:49]: @hmcginnis they are already tested in screener.
